Backpack/CardButton/Classes/SaveCardButton.swift

```diff
@@ -0,0 +1,173 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+public class BPKSaveCardButton: UIButton, CardButtonProtocol {
+
+    public var checked: Bool {
+        get {
+            isSelected
+        }
+        set {
+            isSelected = newValue
+            updateLookAndFeel()
+            onCheckedChange?(checked)
+        }
+    }
```

Comment:
Should we make this private? 
We have the setChecked method to change the checked state of the button.
I am now able to change the checked state by just changing this value, which does not have the animation.

Feel it's ok for us to provide just a single path to change the selection state. 
We probably also want to unhook isSelected, if we are just hooking it up to checked. My original intention was to 'override isSelected' and use that to update everything. 

But considering we want the animation, it's probably for the better to not link checked <> isSelected or any other public variable to maintain selection state.

Reply:
got it. 
- unhook UIButton's isSelected 
- mark var checked to be private (so that there is a single path only)

Reply:
1. 84b32b57ee08c5b61789178e98bf9a502364170f revert isSelected related changes
2. a22ce68f77e803b32c0eed998d6b9088233ff773 public getter, private setter